Jellyfin.Api/Controllers/UserController.cs

```diff
@@ -0,0 +1,554 @@
+﻿#nullable enable
+#pragma warning disable CA1801
```

Comment:
```suggestion
```

Don't need `#nullable enable` (just informed)
Remove pragma in favor of `SuppressMessage` (see https://github.com/jellyfin/jellyfin/issues/2872#issuecomment-617845103 for usage)